### PR TITLE
WOE 5 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/agatha_of_the_vile_cauldron.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Name:Agatha of the Vile Cauldron
+ManaCost:R G
+Types:Legendary Creature Human Warlock
+PT:1/1
+S:Mode$ ReduceCost | ValidCard$ Creature.YouCtrl | Type$ Ability | Amount$ X | MinMana$ 1 | AffectedZone$ Battlefield | Description$ Activated abilities of creatures you control cost X less to activate, where X is CARDNAME's power. This effect can't reduce the mana in that cost to less than one mana.
+A:AB$ PumpAll | Cost$ 4 R G | ValidCards$ Creature.YouCtrl+Other | NumAtt$ +1 | NumDef$ +1 | KW$ Trample & Haste | SpellDescription$ Other creatures you control get +1/+1 and gain trample and haste until end of turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/agatha_of_the_vile_cauldron.txt:6]: StrictlyOther

File: forge-gui/res/cardsfolder/upcoming/agathas_soul_cauldron.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+S:Mode$ Continuous | Affected$ Creature.YouCtrl+counters_GE1_P1P1 | EffectZone$ Battlefield | GainsAbilitiesOf$ Creature.ExiledWithSource | GainsAbilitiesOfZones$ Exile | Description$ Creatures you control with +1/+1 counters on them have all activated abilities of all creature cards exiled with CARDNAME.
+A:AB$ ChangeZone | Cost$ T | Origin$ Graveyard | Destination$ Exile | TgtPrompt$ Choose target card in a graveyard | RememberChanged$ True | ValidTgts$ Card | SubAbility$ TrigImmediateTrig | SpellDescription$ Exile target card from a graveyard. When a creature card is exiled this way, put a +1/+1 counter on target creature you control.
+SVar:TrigImmediateTrig:DB$ ImmediateTrigger | ConditionDefined$ Remembered | ConditionPresent$ Creature | ConditionCompare$ GE1 | Execute$ TrigPutCounter | SubAbility$ DBCleanup | TriggerDescription$ When a creature card is exiled this way, put a +1/+1 counter on target creature you control.
+SVar:TrigPutCounter:DB$ PutCounter | ValidTgts$ Creature.YouCtrl | CounterType$ P1P1 | CounterNum$ 1 | ConditionCompare$ EQ1
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/agathas_soul_cauldron.txt:8]: no condition

File: forge-gui/res/cardsfolder/upcoming/edgewall_inn.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Name:Edgewall Inn
+ManaCost:no cost
+Types:Land
+K:CARDNAME enters the battlefield tapped.
+K:ETBReplacement:Other:ChooseColor
+SVar:ChooseColor:DB$ ChooseColor | Defined$ You | AILogic$ MostProminentInComputerDeck | SpellDescription$ As CARDNAME enters the battlefield, choose a color.
+A:AB$ Mana | Cost$ T | Produced$ Chosen | SpellDescription$ Add one mana of the chosen color.
+A:AB$ ChangeZone | Cost$ 3 T Sac<1/CARDNAME> | TgtPrompt$ Choose target creature card in your graveyard that has an adventure | ValidTgts$ Creature.YouCtrl+AdventureCard | Origin$ Graveyard | Destination$ Hand | SpellDescription$ Return target card that has an Adventure from your graveyard to your hand.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/edgewall_inn.txt:8]: wrong Tgts

**Review by tool4ever — APPROVED** [review]

**Comment by Hanmac** [comment]
Soul Couldron ManaConvert might be problematic because its added as Keyword. not as Hidden Keyword

but for that i made a new Issue #3679

**Pull request merged**